content/post/36-a-look-at-windows.md

```diff
@@ -0,0 +1,52 @@
+---
+date: 1970-01-01
+url: 36-a-look-at-windows
+title: A look at go-ipfs on Windows
+author: Dominic Della Valle
+---
+
+`go-ipfs` is built on top of [Golang](https://golang.org/), which allows for some cross platform compatibility, however it doesn't cover everything. Let's take a look at some of the issues of running `go-ipfs` on Windows and the upcoming solutions for them.
```

Comment:
- Split the first sentence
  - Old: compatibility, however it doesn't
  - New: compatibility. However, it doesn't
- Regarding "upcoming solutions", are these still upcoming or were they in the latest release? It's a bit at odds with the "Now, xx should..." phrasing below. Let's clarify.
- Add one more piece of context. Here's a suggestion, feel free to write your own. "Over the past few weeks, we've fine-tuned several aspects of the Windows experience to fix errors and remove inconsistencies."